apps/examples/aws-example/test/integration/conftest.py

```diff
@@ -0,0 +1,102 @@
+import time
+from datetime import datetime, timezone
+
+import pytest
+from aws_example.model import Something, SomethingParams, Status, StatusType, User
+from hopeit.testing.apps import config
+from moto.server import ThreadedMotoServer
+
+EVENT_TS = datetime.strptime("2020-05-01T00:00:00", "%Y-%m-%dT%H:%M:%S").astimezone(
+    tz=timezone.utc
+)
+
+
+@pytest.fixture
```

Comment:
These fixtures seems to be copy-paste from hopeit.engine. Remove the ones that are not used if that's the case.